content/en/os/latest.markdown

```diff
@@ -2,6 +2,7 @@
 type="docs"
 title="Latest"
 toc_hide=true
+no_version_warning=true
 +++
 
-{{< latest-redirect >}}
\ No newline at end of file
+{{< latest-redirect >}}
```

Comment:
What changed here? Is this github trying to show an added newline at EOF?

Reply:
Correct. There was no EOF after the last curly brace and I was making a linter happy.